config/odp-linux-generic.conf

```diff
@@ -38,3 +38,12 @@ queue_basic: {
 	# Default queue size. Value must be a power of two.
 	default_queue_size = 4096
 }
+
+sched_basic: {
+	# Priority level spread. Each priority level is spread into multiple
+	# scheduler internal queues. A higher spread value typically improves
+	# parallelism and thus is better for high thread counts, but causes
+	# uneven service level for low thread counts. Typically, optimal
+	# value is the number of threads using the scheduler.
```

Comment:
Does this mean that the optimal value is the number of worker threads? In that case, why not make this a default? I thought there was an effective limit on `prio_spread` in that setting it too high would be counterproductive. If an application has 128 threads are you really suggesting that `prio_spread = 128` should be used?

Reply:
*Typically*, the number of threads calling scheduler is optimal. It really needs to be measured in each case. Four seems to be good default for most cases. Current max is 8, so 128 would just error due to bad value.

Application decides number of threads calling scheduler, so implementation cannot "default" to that. Currently, it defaults to 4.